clarifai/client/client.py

```diff
@@ -739,3 +740,75 @@ def _base64_encoded_data_op(self, data, op):
     headers = self._get_json_headers()
     response = self._get_json_response(url, data=data, headers=headers)
     return self._parse_response(response)
+
+  def color(self, files, local_ids=None, meta=None):
+    """Get color from file or list of files.
+
+    The only method used on the file object is read() to get the bytes of the
+    compressed data representation. Ensure that all file objects are pointing
+    to the beginning of a valid data file.
+
+    Args:
+      files: a single (file, name) tuple or a list of (file, name) tuples,
+             where file is an open file-like object containing the encoded data
+             bytes.
+      local_ids: a single string identifier or list of string identifies that
+                 are useful client side. These will be returned in the request
+                 to match up results (even though results to come back in
+                 order).
+      meta: a string of any extra information to accompany the request. This
+            has to be a string, so if passing structured data, pass a
+            json.dumps(meta) string.
+
+    Returns:
+      results: an API reponse including the generated tags. See the docs at
+      https://developer.clarifai.com/docs/ for more detais.
+
+    ## Example:
+    ```
```

Comment:
This is a Markdown syntax, probably we don't want to keep it in the python code.